libphal/phal_dump.C

```diff
@@ -285,8 +291,11 @@ void writeDumpFile(char* data, size_t len, std::filesystem::path& dumpPath)
 }
 
 void collectSBEDump(uint32_t id, uint32_t failingUnit,
-		    const std::filesystem::path& dumpPath)
+		    const std::filesystem::path& dumpPath, const int sbeTypeId)
 {
+    constexpr int ODYSSEY_SBE_DUMP = 0xB;
+    auto forOdySsey = (ODYSSEY_SBE_DUMP == sbeTypeId) ? true : false;
```

Comment:
let us passaround the sbeTypeId

Reply:
Done. Wanted to avoid two extra lines of code in three places but ok.